omnisolver/samplers/van.py

```diff
@@ -0,0 +1,188 @@
+import dimod
+import torch
+import time
+import numpy as np
+from functools import partial
+import random
+from ...van_orig.src_hop_sk.sk import SKModel
+from ...van_orig.src_hop_sk.made2 import MADE
+
+class VanSampler(dimod.Sampler):
+    """Implementatoin of simple random-sampler."""
+
+    def __init__(self, **kwargs):
```

Comment:
I know this is how it's done in original VAN, but what do you think about handling unused keyword args or perhaps changing this to named args completely?

Reply:
It's a stylistic choice, I don't like having more then 3-5 named args.